src/services/token/index.ts

```diff
@@ -0,0 +1,79 @@
+import type { DeleteResult } from 'typeorm';
+import jwt from 'jsonwebtoken';
+import { dbInstanse } from '../db/utils/getConnection';
+import Token from '../db/entities/Token';
+import User from '../db/entities/User';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require('dotenv').config();
+
+type GenerateTokens = {
+  accessToken: string;
+  refreshToken: string;
+};
+
+class TokenService {
```

Comment:
you don't need to name your methods as `methodnameTokens` as they're already a part of TokenService
please remove "Tokens" part from methods names

Reply:
generateTokens => generate
etc.